articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -146,19 +151,21 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-For more information on storing and saving the `access_token` to use later when calling the API, see the [Storing Tokens step](/quickstart/webapp/aspnet-core/v2/03-storing-tokens).
+For more information on saving and storing the access token, see the [Storing Tokens](/quickstart/webapp/aspnet-core/v2/03-storing-tokens) step.
+
+For general information on using APIs with web applications, see the [Calling APIs from Server-side Web Apps](/api-auth/grant/authorization-code) article.
 
-For general information on using APIs with web applications, please read [Calling APIs from Server-side Web Apps](/api-auth/grant/authorization-code).
+## Add the `Login` and `Logout` Methods
 
-## Add Login and Logout Methods
+Add the `Login` and `Logout` actions to `AccountController`. 
 
-Next, you will need to add `Login` and `Logout` actions to the `AccountController`. 
+To add the `Login` action, call `ChallengeAsync` and pass "Auth0" as the authentication scheme. This will invoke the OIDC authentication handler you registered in the `ConfigureServices` method.
 
-For the Login, you will need to call `ChallengeAsync`, passing "Auth0" as the authentication scheme which will be challenged. This will invoke the OIDC authentication handler you registered in the `ConfigureServices` method.
+After the OIDC middleware signs the user in, the user is also automatically signed in to the cookie middleware. This allows the user to be authenticated later. 
```

Comment:
> This allows the user to be authenticated later

Please keep this as 

> This allows the user to be authenticated on subsequent requests.

The way you worded it makes it sounds like when they come back at a later stage. But this is not the case. It should be made clear the all subsequent requests which are part of the current session will be authenticated by the cookie middleware.